scriptmodules/emulators/ti99sim.sh

```diff
@@ -36,7 +36,7 @@ function install_ti99sim() {
 function configure_ti99sim() {
     mkRomDir "ti99"
     moveConfigDir "$home/.ti99sim" "$md_conf_root/ti99/"
+    ln -sf "$biosdir/TI-994A.ctg" "$md_inst/TI-994A.ctg"
```

Comment:
yeah, that can be changed too. actually, dont worry, Ill merge as is my brain is still not functioning correctly after high fever for 3 days and I will revisit when I can think clearly (still have a fever - should be in bed!)